pkg/plugins/golang/v3/init.go

```diff
@@ -171,12 +172,23 @@ func checkDir() error {
 			if strings.HasPrefix(info.Name(), ".") {
 				return nil
 			}
+			// Allow files ending with '.md' extension
+			if strings.HasSuffix(info.Name(), ".md") && !info.IsDir() {
+				return nil
+			}
+			// Allow capitalized files (like LICENSE, CODEOWNERS..)
+			for index, l := range info.Name() {
+				if !unicode.IsUpper(l) || info.Name() == "PROJECT" {
```

Comment:
Any way, checking the name against `PROJECT` should not be done inside of the loop.

Reply:
Fixed!